py-polars/polars/io/database/_arrow_registry.py

```diff
@@ -41,8 +41,7 @@ class ArrowDriverProperties(TypedDict):
         "repeat_batch_calls": False,
     },
     "kuzu": {
-        # 'get_as_arrow' currently takes a mandatory chunk size
-        "fetch_all": ("get_as_arrow", 10_000),
+        "fetch_all": "get_as_pl",
```

Comment:
Nice! The type for this field can now be changed back to `str`, right?

Reply:
Sure; always worth being tidy 😎 
Can reinstate later if we come across a similar case 👌